makefiles/Mf-install.in

```diff
@@ -115,14 +115,14 @@ libbininstall: ${LibBin}
 	$I -m 444 ${PetiteBoot} ${LibBin}/petite.boot
 	if [ "${InstallPetiteName}" != "petite" ]; then\
           /bin/rm -f ${LibBin}/${InstallPetiteName}.boot;\
-          ln -f ${LibBin}/petite.boot ${LibBin}/${InstallPetiteName}.boot;\
+          ln -f ${InstallLibBin}/petite.boot ${LibBin}/${InstallPetiteName}.boot;\
         fi
 	$I -m 444 ${SchemeBoot} ${LibBin}/scheme.boot;\
 	if [ "${InstallSchemeName}" != "scheme" ]; then\
           /bin/rm -f ${LibBin}/${InstallSchemeName}.boot;\
-          ln -f ${LibBin}/scheme.boot ${LibBin}/${InstallSchemeName}.boot;\
+          ln -f ${InstallLibBin}/scheme.boot ${LibBin}/${InstallSchemeName}.boot;\
         fi
-	ln -sf ${LibBin}/scheme.boot ${LibBin}/${InstallScriptName}.boot;
```

Comment:
This appears to be the only symbolic link. Is there a reason not to use a hard link here? Or to use a relative symbolic link?